scripts/bravo_csv.py

```diff
@@ -214,6 +214,23 @@ def compute_transfer_volume(currentStep, lims, log):
     return returndata
 
 
+def aliquot_fixed_volume(currentStep, lims, volume, log):
```

Comment:
The log does not seem to be used in the function, is it needed?

Reply:
Hi Anand, yes, I also noticed it. Because I don't really know which type of error may occur when querying those variables. Thus I kept the log as placeholder, and it can be used in case we meet any in the fugure.